### ISSUE TITLE
zsh completion suggests flags but not files for `edit` subcommand

### ISSUE THREAD

**Issue by roobre** [twpayne/chezmoi#558]
**Describe the bug**
On zsh, triggering tab-completion like `[roobre@Archiroo]% ~> chezmoi edit<tab><tab>` shows:

```
[roobre@Archiroo]% ~> chezmoi edit -
--apply        -a  -- apply edit after editing                                                                       
--color            -- colorize diffs                                                                                 
--config       -c  -- config file                                                                                    
--debug            -- write debug logs                                                                               
--destination  -D  -- destination directory                                                                          
--diff         -d  -- print diff after editing                                                                       
--dry-run      -n  -- dry run                                                                                        
--follow           -- follow symlinks                                                                                
--prompt       -p  -- prompt before applying (implies --diff)                                                        
--remove           -- remove targets                                                                                 
--source       -S  -- source directory                                                                               
--verbose      -v  -- verbose                                                                                        
```

If I trigger it like `[roobre@Archiroo]% ~> chezmoi edit ./<tab><tab>`, I don't get anything.

**To reproduce**
See description.

**Expected behavior**
I'd expect `chezmoi edit` to suggest paths by default, and flags if a `-` is the last input character before pressing tab, which is like completion works for coreutils like `cp` or `mv`.

**Output of command with `--verbose`**
*Does not apply*

**Output of `chezmoi doctor`**
```
$ chezmoi doctor
warning: version 1.7.11
     ok: runtime.GOOS linux, runtime.GOARCH amd64
     ok: /home/roobre/.local/share/chezmoi (source directory, perm 700)
     ok: /home/roobre (destination directory, perm 700)
warning: /home/roobre/.config/chezmoi/chezmoi.toml (configuration file)
     ok: /bin/zsh (shell)
     ok: /usr/bin/nano (editor)
     ok: /usr/bin/vimdiff (merge command)
     ok: /usr/bin/git (source VCS command, version 2.24.1)
     ok: /usr/bin/gpg (GnuPG, version 2.2.19)
warning: op (1Password CLI, not found)
warning: bw (Bitwarden CLI, not found)
warning: gopass (gopass CLI, not found)
     ok: /usr/bin/keepassxc-cli (KeePassXC CLI, version 2.5.2)
warning: lpass (LastPass CLI, not found)
warning: pass (pass CLI, not found)
warning: vault (Vault CLI, not found)
```

**Comment by twpayne** [comment]
Thanks for reporting this :) I didn't realize that I had to call a couple of extra functions to enable zsh completion for positional arguments. This should all be fixed with #559.

**Comment by roobre** [comment]
I just tested it with `master` and it works wonders, thank you!
